css/main.css

```diff
@@ -1,8 +1,17 @@
+:root {
+  --color-text: #fff;
+  --color-background: black;
+  --base: 20px;
```

Comment:
I put “base” because it sets the “font-size” property on the html. So that means anything that uses the “rem” unit changes based on what “base” is.

So if one wanted to set the margins in a way that scales, they could set it to “2rem”, and it would increase if “base” were to increase

Reply:
I'd keep it simple on the first PR and just make it font-size-normal and only use it for that. All other stuff we can add later :-)